tests/test_models/test_nn/test_rnn.py

```diff
@@ -45,10 +48,37 @@ def test_rnn_model_run_weekly_overfit_with_scaler(ts_dataset_weekly_function_wit
     assert mae(ts_test, future) < 0.06
 
 
+@pytest.mark.parametrize(
+    "embedding_sizes,features_to_encode",
+    [({}, []), ({"categ_regr_label": (2, 5), "categ_regr_new_label": (1, 5)}, ["categ_regr", "categ_regr_new"])],
+)
+def test_rnn_backtest(ts_different_regressors, embedding_sizes, features_to_encode):
```

Comment:
Are we sure this test is about backtest and not about handling categoricals?

Reply:
I was surprised that we don't have backtest test, so I added it to check whether it works in all main setups